### PR TITLE
Focus input field after detail is shown.

### PR DESCRIPTION
**Pull request by clemenshelm**

### CODE REVIEW

File: client/bills.coffee [hunk 1]
```diff
@@ -1,4 +1,13 @@
 Template.bills.events
+  "click .detail-link": (e) ->
+    link = $(e.target)
+    detail = $ link.attr('href')
+    # Without the timeout the input would be focused before the details are shown.
+    # When details are shown, focus gets lost.
+    setTimeout ->
```
Comment (DominikGuzei) [inline client/bills.coffee]: I would say this is already the first indication that you should move the details view into its own template and only show the one that is active :wink: There is a Blaze hook that you can use then e.g.: `Template.bill_details.rendered = -> @$('.field input').first().focus()`

Comment (clemenshelm) [inline client/bills.coffee]: Thanks, I'll do that! :smile_cat:

**Comment by clemenshelm** [comment]
Moved the details into their own template. Please review :)

**Comment by DominikGuzei** [comment]
Much better now! :+1:

**Comment by clemenshelm** [comment]
Cool, then I'll merge it! :)

**Pull request merged**